packages/python/test/fixtures/28-pydantic/requirements.txt

```
@@ -1,2 +1,2 @@
-fastapi==0.54.1
+fastapi==0.105.1
```

```suggestion
fastapi==0.105.0
```

Did you have a typo?

https://pypi.org/project/fastapi/#history

---

Wow yeah I think that's it. 0.105.0 is the version in the Linear ticket too. Good catch!